ios/Podfile

```diff
@@ -8,10 +8,11 @@ target 'SampleApp' do
   use_unimodules!
   config = use_native_modules!
 
-  use_react_native!(:path => config["reactNativePath"])
+  use_react_native!(:path => config["reactNativePath"])  
   pod 'RCT-Folly', :podspec => '../node_modules/react-native/third-party-podspecs/RCT-Folly.podspec'
   pod 'boost', :podspec => '../node_modules/react-native/third-party-podspecs/boost.podspec'
-  pod 'CustomerIOMessagingPushAPN', :podspec => 'https://raw.githubusercontent.com/customerio/customerio-ios/1.1.1/CustomerIOMessagingPushAPN.podspec'
+  pod 'CustomerIO/Tracking', '~> 1.1'
```

Comment:
This is not required. This might result into a conflict. We do not have to add `'CustomerIO/Tracking'` using pods because this is being handled by the our react native package.

Only `CustomerIOMessagingPushAPN` or `CustomerIOMessagingPushFCM` has to be added explicitly.

Reply:
Because the Tracking module is a dependency of the messaging push modules, technically the tracking module is always installed alongside the messaging push module. 

Me adding the tracking module to this file is a personal preference. I find it easier to read when child dependencies are listed like this. Helps understand what is and isn't installed.

Reply:
I understand that it makes it easier to read but the relevant SDK already exists in the project and is installed as soon as the user does `npm install customerio-reactnative`.